vcpkg/maintainers/publishing-to-a-git-registry.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Publishing to a Git registry
+description: Learn to publish vcpkg ports to a Git-based registry
+author: vicroms
+ms.author: viromer
+ms.date: 7/21/2023
+ms.prod: vcpkg
+---
+# Publishing to a Git registry
+
+## Step 1: Create a Git registry
+
+The first step to create a Git-based registry is to create a Git repository:
+
+```Console
+mkdir vcpkg-registry
+cd vcpkg-registry
+git init
+```
+
+## Step 2: Create the registry folders
+
+Git-based registries are actually just Git repositories with a specific layout that vcpkg expects.
+A vcpkg Git-based registry requires the following:
+
+* a folder named `ports` to contain the registry's ports,
+* a folder named `versions` to contain the versions database,
+* a file named `baseline.json`  in the `versions` folder
+
```

Comment:
I'm not sure this is the right place for this kind of information, but would it be appropriate to provide a brief explanation of why these are required/important before diving into how to create them?